config/routes/dev/routes.yaml

```diff
@@ -1,10 +1,5 @@
-_wdt:
-    resource: "@WebProfilerBundle/Resources/config/routing/wdt.xml"
-    prefix: /_wdt
-
-_profiler:
-    resource: "@WebProfilerBundle/Resources/config/routing/profiler.xml"
-    prefix: /_profiler
+WebProfilerBundle:
```

Comment:
using the bundle name here as done in the recipe, I applied this pattern for other routes move, as there is no chances that the bundle name is used as a route name, so no conflicts